packages/snaps-sdk/src/jsx/components/form/Dropdown.ts

```diff
@@ -0,0 +1,41 @@
+import type { MaybeArray } from '../../component';
+import { createSnapComponent } from '../../component';
+import type { DropdownOptionElement } from './DropdownOption';
+
+/**
+ * The props of the {@link Dropdown} component.
+ *
+ * @property name - The name of the dropdown. This is used to identify the
+ * state in the form data.
+ */
+type DropdownProps = {
```

Comment:
Maybe we should support a `placeholder` and default option here?

Reply:
Default option is the same as using `value`. The underlying DS component doesn't support a `placeholder` so I would vote we put that off